sandbox-test/src/test/scala/scala/pickling/PicklingProtocol.scala

```diff
@@ -0,0 +1,20 @@
+package scala.pickling
+
+import scala.pickling.binary.BinaryFormats
+import scala.pickling.internal.HybridRuntime
+import scala.pickling.json.JsonFormats
+import scala.pickling.pickler.AllPicklers
+
+trait PicklingProtocol extends {
+  val oldRuntime = internal.currentRuntime
+  val onlyLookup = {
+    val currentRuntime =
+      if(oldRuntime.isInstanceOf[HybridRuntime]) oldRuntime
```

Comment:
Actually, I haven't tested it yet, since we're using only `JsonPicklingProtocol`. Any special reason why this wouldn't work?